runtime/onert/backend/train/KernelGenerator.cc

```diff
@@ -257,9 +256,11 @@ void KernelGenerator::visit(const ir::train::operation::Loss &node)
     }
     case ir::train::LossCode::CategoricalCrossentropy:
     {
+      auto loss_info =
+        reinterpret_cast<const ir::train::LossCategoricalCrossentropyInfo &>(node.param());
```

Comment:
```suggestion
        nnfw::misc::polymorphic_downcast<const ir::train::LossCategoricalCrossentropyInfo &>(node.param());
```

Reply:
I tried to this downcast function but it occured `source type is not polymorphic` error. I didn't think about it much. I'll take a closer look.